### ISSUE TITLE
[CTI] The ticket title is not automatically filled when opening the "new ticket screen" if the caller is unknown

### ISSUE THREAD

**Issue by martini** [zammad/zammad#5082]
### Used Zammad Version

6.2

### Environment

- Installation method: package
- Operating system (if you're unsure: `cat /etc/os-release` ): any
- Database + version:   any
- Elasticsearch version:  any
- Browser + version: any

### Actual behaviour

Prerequisite: Enabled and configured CTI integration:
 
The ticket title is not automatically filled when opening the "new ticket screen" if the caller is unknown (if caller is know, all looks ok)

![test1](https://github.com/zammad/zammad/assets/110226/0b2eca58-6f40-4ca8-8528-22d4e1c7cd1d)

### Expected behaviour

The ticket title should be automatically filled when opening the "new ticket screen" if the caller is unknown.

![test2](https://github.com/zammad/zammad/assets/110226/e733bdc5-30d6-43c3-96d6-69487fc42424)

### Steps to reproduce the behaviour

* Enabled and configured CTI integration
* create an agent with answering number
* answer an incoming customer call (THIS IS IMPORTANT: **from an unknown customer**) by this agent
* new ticket screen will get opened automatically, but no title is pre-filled

### I'm sure this is a bug and no feature request or a general question.

yes
